v3/src/components/graph/components/inspector-panel/hide-show-menu-list.tsx

```diff
@@ -0,0 +1,52 @@
+import { MenuItem, MenuList, useToast } from "@chakra-ui/react"
+import React from "react"
+import { useDataSetContext } from "../../../../hooks/use-data-set-context"
+import t from "../../../../utilities/translation/translate"
+
+interface IProps {
+  showParentToggles?: boolean
+  setShowParentToggles?: (show: boolean) => void
+  showMeasuresForSelection?: boolean
+  setShowMeasuresForSelection?: (show: boolean) => void
+}
+
+export const HideShowMenuList = ({showParentToggles, setShowParentToggles, showMeasuresForSelection,
+    setShowMeasuresForSelection}: IProps) => {
+  const data = useDataSetContext()
+  const toast = useToast()
+
+  const handleMenuItemClick = (menuItem: string) => {
+    toast({
+      title: 'Menu item clicked',
+      description: `You clicked on ${menuItem}`,
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    })
+  }
+
+  const hideSelectedString = data?.selectCases && data?.selectCases.length > 1
+                              ? t("DG.DataDisplayMenu.hideSelectedPlural")
+                              : t("DG.DataDisplayMenu.hideSelectedSing")
```

Comment:
The default in case of no `data` should be the plural:
```suggestion
  const hideSelectedString = (data?.selectCases?.length ?? 0) === 1
                              ? t("DG.DataDisplayMenu.hideSelectedSing")
                              : t("DG.DataDisplayMenu.hideSelectedPlural")
```